src/components/app/details/triggerView/triggerViewConfigDiff/TriggerViewConfigDiff.tsx

```diff
@@ -286,16 +305,20 @@ export default function TriggerViewConfigDiff({
 
     const renderConfigDiffViewHeader = () => {
         return (
-            <div className="trigger-view-config-diff__tabs dc__border-bottom">
-                <div className="fs-13 fw-6 lh-20 cn-9 bcn-0 m-0 pt-12 pb-12 pl-16 pr-16 dc__border-right">
+            <div className="trigger-view-config-diff__tabs bcn-0 dc__border-bottom">
+                <div className="fs-13 fw-6 lh-20 cn-9 m-0 pt-12 pb-12 pl-16 pr-16 dc__border-right">
                     Deployment Configuration
                 </div>
-                <div className="fs-13 fw-4 lh-20 pt-12 pb-12 pl-16 pr-16 cn-9 bcn-0 dc__border-right">
-                    Last Deployed Configuration
-                </div>
-                <div className="flex left bcn-0">
+                {isRecentConfigAvailable ? (
```

Comment:
can we leverage && here instead of ternary?

Reply:
done